packages/blockchain-wallet-v4/src/redux/data/bch/reducers.js

```diff
@@ -87,6 +89,24 @@ const bchReducer = (state = INITIAL_STATE, action) => {
     case AT.FETCH_BCH_TRANSACTION_HISTORY_FAILURE: {
       return assoc('transaction_history', Remote.Failure(payload), state)
     }
+    case AT.FETCH_BCH_SPENDABLE_BALANCE_LOADING: {
+      return state
```

Comment:
Why isn't this  "return assoc('spendable_balance', Remote.Loading, state)"?

Reply:
Because of this:

https://github.com/blockchain/blockchain-wallet-v4-frontend/blob/release/v4.0.0-alpha10/packages/blockchain-wallet-v4-frontend/src/layouts/Wallet/MenuTop/Balance/TotalBalance/selectors.js#L7-L8

If TotalBalance component doesn't know the btc/bch/eth balances it guesses 0 while it waits. So if we set spendableBalance to Loading while its loading the user sees 0 instead of their previous balance before the fetch was called.

Real world scenario: Send some of my btc -> update spendable balance -> see balance go to 0 for a second -> balance updates.

This makes the flow: send some of my btc -> update spendable balance